app/views/uploaded_files/_form.html.erb

```diff
@@ -75,7 +75,7 @@
 
       <div class="form-group">
         <%= form.label :link_Name %>
-        <%= form.text_field :link_name, :class => "form-control", id:"input-upload-link-name", required: true %>
+        <%= form.text_field :link_name, :class => "form-control", id:"input-upload-link-name", placeholder: "optional file name" %>
```

Comment:
placeholders should contain valid examples, description for inputs should be described in the label for accessibility reasons:
https://www.webaccessibility.com/best_practices.php?best_practice_id=2440